Source/WebCore/platform/graphics/gstreamer/eme/WebKitOpenCDMDecryptorGStreamer.cpp

```diff
@@ -211,7 +203,7 @@ static bool webKitMediaOpenCDMDecryptorDecrypt(WebKitMediaCommonEncryptionDecryp
 
     // Decrypt cipher.
     GST_TRACE_OBJECT(self, "decrypting");
-    if (int errorCode = adapter_session_decrypt(priv->m_openCdm.get(), reinterpret_cast<void*>(buffer), reinterpret_cast<void*>(subSamplesBuffer), subSampleCount, mappedIV.data(), static_cast<uint32_t>(mappedIV.size()), mappedKeyID.data(), mappedKeyID.size())) {
+    if (int errorCode = opencdm_gstreamer_session_decrypt(priv->m_openCdm.get(), buffer, subSamplesBuffer, subSampleCount, ivBuffer, keyIDBuffer, 0)) {
```

Comment:
What's the trailing 0?

Reply:
An additional param added by netflix (uint32_t initwithLast15). Irrelevant for webkit and 0 shouldbe used in that case.

Reply:
I recommend refactoring this interface to take a struct parameter, so that random bits of information like that can be properly named and more easily read by maintainers, but that's you so I guess your choice ;)

Reply:
It's now slightly better then before (due to reduction of number of params, and dropping cast). Unfortunately this time it was a coordinated with netflix guys (they had to update too) and I didn't want to force them to do another round.

Reply:
I'd say this is a case of "Short-term pain, long-term gain."